azure-go-sdk-conceptual/TOC.yml

```diff
@@ -0,0 +1,25 @@
+items:
+  - name: Azure SDK for Go
+    href: index.yml
+  - name: Overview
+    items:
+    - name: Installation
+      href: azure-sdk-go-install.md
+    - name: Tools
+      href: azure-sdk-go-tools.md
+  - name: Quickstarts
+    items:
+    - name: Deploy a virtual machine
```

Comment:
"Programmatically deploy a Virtual Machine" - make it clear this is an example of using the SDK, not of using Azure VMs with Go

Reply:
We are not allowed to have titles which word-wrap in the ToC. The fact that the VM is deployed with the SDK should be clear from context. None of the other quickstarts have this naming convention, either.

Instead the section name will be changed to "Go Quickstarts". See the [python](https://docs.microsoft.com/en-us/python/azure/?view=azure-python), [java](https://docs.microsoft.com/en-us/java/azure/), [.net](https://docs.microsoft.com/en-us/dotnet/azure/) SDK pages.

Reply:
I think that would make it worse. The quickstarts you linked in the examples are about consuming Azure services, not the SDK.